### PR TITLE
Added-eye-icon-button-on-registration-Page

### PR DESCRIPTION
**Pull request by arbazkhan56**
I am adding an icon on the registration page. I hope you Like it. And your design is very good looking. I Like it.

### CODE REVIEW

**Comment by annu12340** [comment]
Hey thanks for the pr
I liked that icon 😄

File: templates/register.html [hunk 1]
```diff
@@ -1,64 +1,101 @@
...
+
+<head>
+    <meta charset="UTF-8">
+    <title>Title</title>
+    <link rel="stylesheet" href="/static/base.css">
+    <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.3.1/css/bootstrap.min.css" integrity="sha384-ggOyR0iXCbMQv3Xipma34MD+dH/1fQ784/j6cY/iJTQUOhcWr7x9JvoRxT2MZw1T" crossorigin="anonymous">
+
+    <link rel="stylesheet" type="text/css" href="https://code.ionicframework.com/ionicons/2.0.1/css/ionicons.min.css">
+    <style>
+    	.icon-wrapper{
```
Comment (annu12340) [inline templates/register.html]: Add the css inside **static** folder

**Review by annu12340 — CHANGES_REQUESTED** [review]

File: templates/register.html [hunk 2]
```diff
@@ -1,64 +1,101 @@
...
+            <div class="col-7"><img src="/static/registerPg_Finance.png" alt="" width="760" height="450"></div>
+        </div>
+
+        <div class="footer">
+            <p class="text-center"> Finance tracker</p>
+        </div>
+
+</body>
+<script>
+    	function myFunction() {
```
Comment (annu12340) [inline templates/register.html]: And change the function name from **myFunction()** to something very specific and relevant

**Review by annu12340 — CHANGES_REQUESTED** [review]

**Comment by arbazkhan56** [comment]
> Hey thanks for the pr
> I liked that icon 😄

Thank you. I am changing it according to you.

**Comment by annu12340** [comment]
Pls resolve the conflicts

**Pull request merged**